src/many-kvstore/src/storage.rs

```diff
@@ -87,7 +87,15 @@ impl KvStoreStorage {
             u64::from_be_bytes(bytes)
         });
 
-        let latest_event_id = EventId::from(height << HEIGHT_EVENTID_SHIFT);
+        // The call to `saturating_sub()` is required to fix
```

Comment:
Can we just do the proper fix for kvstore since we haven't started a production network yet? Just store the `latest_event_id` in the database on `commit()` and +1 it every event.

Reply:
Yes of course